.nvmrc

```
@@ -0,0 +1 @@
+v10.18.1
```

Nit: Elsewhere we've been less-specific with the Node.jsversion we're using (i.e. just using `v10` instead of `v10.18.1`). We're more specific in the extension because there are a couple of features we use not in earlier versions of Node.js.

Maybe we should use `v10` here as well, to be consistent?

```suggestion
v10
```